### DISCUSSION TITLE
JVM metrics

### DISCUSSION THREAD

**Discussion by yuyu333333** [apache/pulsar#20215] [Q&A] [answered]
In the 'pulsar metrics' section of the pulsar official website, I have seen many indicators, but I cannot distinguish which are JVM related indicators; Or is there a separate organization of JVM metrics? I would like to know which ones are JVM metrics

**Comment by AlvaroStream** [comment]
Hello, 

that is not documented there maybe we can open an issue to include them.

But you can see an example [here](https://github.com/apache/pulsar/blob/a22700df665a9ea0a8db98334b6d4337e5e490f3/pulsar-broker/src/test/resources/prometheus_metrics_sample.txt) and you can see the information for [bookeeper](https://github.com/apache/pulsar/blob/a22700df665a9ea0a8db98334b6d4337e5e490f3/grafana/dashboards/bookkeeper.json) of [zookeeper](https://github.com/apache/pulsar/blob/a22700df665a9ea0a8db98334b6d4337e5e490f3/grafana/dashboards/zookeeper.json#L8719)

**Comment by asafm** [comment, marked as answer]
I created a new PR for the documentation containing the missing JVM metrics.
https://github.com/apache/pulsar-site/pull/567/files

**Reply by asafm** [reply]
@yuyu333333 Can you mark an answer?

**Reply by asafm** [reply]
@yuyu333333

**Reply by tisonkun** [reply]
Thank you @asafm!
